webpack.config.js

```diff
@@ -18,7 +18,12 @@ const commonConfig = {
                     presets: [
                         [
                             require.resolve('babel-preset-env'),
-                            { modules: false }
+                            {
+                                modules: false,
+                                targets: {
+                                    electron: '2.0.3'
```

Comment:
can we say something like >= 2 here?

Reply:
@saghul 
Nops. It throws an error. I already tried '>=2.0.0' and '2.0.x' too. 
It says 'Versions must be in semver format (major.minor.patch).
IDK why doesn't it take those as valid.

I tried using `process.versions.electron` too but the electron is not initialised at that point.

Should I require `package.json` as JSON object if you want it to be automatic?

Reply:
> Should I require package.json as JSON object if you want it to be automatic?

Yeah, that sounds good!

Reply:
Done.